src/Compilers/CSharp/Test/Symbol/Symbols/ReplaceOriginalTests.cs

```diff
@@ -1,29 +1,26 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
-using System.Linq;
-using System.Threading;
 using Microsoft.CodeAnalysis.CodeGen;
 using Microsoft.CodeAnalysis.CSharp.Symbols;
 using Microsoft.CodeAnalysis.CSharp.Syntax;
 using Microsoft.CodeAnalysis.CSharp.Test.Utilities;
 using Microsoft.CodeAnalysis.Semantics;
+using Microsoft.CodeAnalysis.Test.Utilities;
 using Roslyn.Test.Utilities;
+using System.Linq;
+using System.Threading;
```

Comment:
I personally do not care whichever way usings are sorted
I just use the built-in   "Sort Using" feature.

If there is a discussion about this, I'd strongly support following the default behavior :-)